lightwood/mixers/helpers/default_net.py

```diff
@@ -34,11 +36,21 @@ def __init__(self, dynamic_parameters,
         super(DefaultNet, self).__init__()
 
         if shape is None and pretrained_net is None:
-            shape = [self.input_size, max([self.input_size*2,self.output_size*2,400]), self.output_size]
+            # default shape, inflated, yields 240k params if input has 200 components
+            shape = [self.input_size, max([self.input_size*2, self.output_size*2, 400]), self.output_size]
+
+            # if NN is too big, we do not inflate and aim for max target_params
+            if reduce(lambda x, y: x*y, shape) > target_params:
+                hidden_size = int(target_params//(self.input_size*self.output_size))
```

Comment:
I had actually forgotten this was a thing, can you replace with `math.floor` after the division? Unless you think this operator is common knowledge, might trip up people.

Reply:
sure thing